File: Php/poo/salary/library/health.php
<?php

    class health extends person {
    private $valueHealth;
    
    public function __construct($valueHealth) {
        $this->valueHealth = $valueHealth;
    }

    public function getValueHealth(){
        return $this->valueHealth;
    }

    public function calculateDiscount(): float {
        $this->calculate = ($this->getValueHealth() * 0.04);
        return $this->calculate;
    }
}

File: Java/poo/salary/library/health.java
package poo.salary.library;

public class health {
    private double valueHealth;

    public health(double valueHealth) {
        this.valueHealth = valueHealth;
    }

    public double getValueHealth() {
        return this.valueHealth;
    }

    public double calculateDiscount() {
        return (this.getValueHealth() * 0.04);
    }
}